section3/reflection.md

```diff
@@ -1,17 +1,52 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+* There were a couple times where it helped me step away and come back with more focus.
+* Other times, I was unable to use the technique because I would get annoyed by the idea of getting distracted while in the middle of a project.
+* I am not sure this technique is for me, as I will be focused for a long period of time, but I can get distracted or pulled away for more than 3 minutes too easily - cooking, dog, cleaning etc.
 
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
+* It went ok. I had to stop using pom because i felt it was interrupting my progress or focus.
```

Comment:
I definitely understand. We tend to use poms a lot in the classroom setting, so you'll get some more exposure. But definitely do what works for you in your personal and group work time.